### ISSUE TITLE
Suddently, not scanning

### ISSUE THREAD

**Issue by SteveP-UK** [sneakypete81/updatescanner#304]
Hi Sneaky Pete

Long time user of your excellent add-on, but today, I hit a problem.

My PC updated to Firefox 64 overnight. Maybe coincidentally, my normal initial daily Update Scanner scan didn't find any changes in my stored pages. Bit weird (as I have so many).

I tried scanning the pages individually by right clicking on a sample of my stored pages in US, but it still said it couldn't find any changes. I ran Update scanner on another PC that has the identical set of pages and it worked perfectly.

So, I've concluded something is wrong on this PC. I've uninstalled/reinstalled US three times now and it hasn't made a difference.

I've deleted all my stored pages manually (after backing them up) and tried restoring them. Pages restore OK but US still says no changes on any of the sites when US is run.

I'm 100% sure the exported file of sites is robust (as it works on my other PC).

I've 'started again' with a fresh install of US and begun to add pages in (so I don't rely on the backup file). It still doesn't work.

I guess I'm out of ideas now. Is there anything else I can try?

I'm pretty competent on a PC and very used t trouble-shooting issues - but this one, I think, has got me beat :) so any advice is welcome.

Thanks

Steve

#### Steps to reproduce the problem:
[Include webpage URL, if applicable]

#### What actually happened:
[Include a screenshot]

**Comment by SteveP-UK** [comment]
Cheers! A good suggestion.

It's all very weird, as your addin is normally 100% robust and just 'gets
on with the job'.

I'd avoided setting up a new profile because I didn't want to muck up any
other settings for other installed addins.

Can you remind me how to set up a new FF profile without it impacting on
other addins... Or, have I misunderstood the profile 'thing'?

Regards

Steve

**Comment by sneakypete81** [comment]
Each profile is like a fresh browser install, with its own add-ons, settings, history, bookmarks, etc. See the link above for details.

**Comment by SteveP-UK** [comment]
Couldn't see the link you mentioned... 😊

**Comment by sneakypete81** [comment]
https://support.mozilla.org/kb/profile-manager-create-and-remove-firefox-profiles

**Comment by SteveP-UK** [comment]
Cheers! I'm working away for the next few days (without the affected PC). I'll report back when I'm next in front of it (probably Tuesday)

Thanks for your help so far

Steve

**Comment by SteveP-UK** [comment]
Hi Sneaky Pete. Great news! It's all working fine now - and I didn't need to muck about with the profiles. So here's a bit of a back story. I've been working away for a few days with*out* the affected PC. My laptop was using US without any problems. But then, just like the PC at home, FF updated on the Laptop and then US just wouldn't find any changes to websites... It seemed to be running OK, but just didn't 'see' any changes, even on sites I know change every hour. What I learned was that leaving it all for a few hours with FF running/open seemed to sort out the problem. Additionally, when I returned home today, I was pleased to see that US was now working perfectly on the *home PC* too (and it hadn't been when I left three days ago)

So, in short, I seem to experience a temporary problem after a FF update that resolved itself in 24 hours. Not sure if that makes nay sort of technical sense... or even *any sort* of sense :)... but normality has returned on all my devices. Keep up the good work and Merry Christmas!

Cheers
Steve
